### PR TITLE
delete trailing space and ineffectual assignment

### PR DESCRIPTION
**Pull request by yu8833**
Signed-off-by: yupengzte <yu.peng36@zte.com.cn>

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by cblecker** [comment]
@k8s-bot ok to test

**Comment by cblecker** [comment]
@k8s-bot bazel test this
@k8s-bot unit test this

File: pkg/api/service/util_test.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ func TestGetLoadBalancerSourceRanges(t *testing.T) {
 		annotations[AnnotationLoadBalancerSourceRangesKey] = v
 		svc := api.Service{}
 		svc.Annotations = annotations
-		cidrs, err := GetLoadBalancerSourceRanges(&svc)
+		_, err := GetLoadBalancerSourceRanges(&svc)
```
Comment (k82cn) [inline pkg/api/service/util_test.go]: we can not remove this according to CI's report.

**Comment by YuPengZTE** [comment]
Thanks @k82cn.

**Comment by k82cn** [comment]
@k8s-bot bazel test this

**Comment by k82cn** [comment]
/lgtm

minor changes

**Comment by YuPengZTE** [comment]
@k8s-bot test this

**Pull request closed without merging**